tests/fiaas_deploy_daemon/deployer/kubernetes/test_autoscaler.py

```diff
@@ -31,20 +31,20 @@ def test_default_spec_should_create_no_autoscaler(app_spec):
 
 
 def test_autoscaler_enabled_and_1_replica_gives_no_autoscaler(app_spec):
-    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=2, cpu_threshold_percentage=50))
+    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=1, max_replicas=2, cpu_threshold_percentage=50))
     app_spec = app_spec._replace(replicas=1)
     assert should_have_autoscaler(app_spec) is False
 
 
 def test_autoscaler_enabled_and_2_replica_and_no_requested_cpu_gives_no_autoscaler(app_spec):
-    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=2, cpu_threshold_percentage=50))
+    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=2, max_replicas=2, cpu_threshold_percentage=50))
```

Comment:
Similar here, setting min=max means there shouldn't be an autoscaler, so this isn't making it test the right property of the condition